gradle.properties

```
@@ -16,7 +16,7 @@
 # under the License.
 
 group=org.apache.cassandra.spark
-version=1.0.0
+version=1.0.0.36
```

this shouldn't change
```suggestion
version=1.0.0
```